### PR TITLE
Arsenal - Fix animations not playing properly

### PR DESCRIPTION
**Pull request by johnb432**
**When merged this pull request will:**
- Add common `FUNC(stopGesture)`. [Credit: ACRE mod](https://github.com/IDI-Systems/acre2/blob/master/addons/sys_gestures/fnc_stopGesture.sqf)
- Prevent losing magazines by opening the arsenal while reloading.
- Fix floating weapons/misaligned grips while browsing.
- Make arsenal open left panel for the currently selected weapon.
- Fix missing `removeVest` command in VR Arsenal.
- Fix weapon animations misbehaving when opening the arsenal in the middle of a weapon switch.
- Repro steps: Switch weapons and open the arsenal. You can observe the animations misbehaving.

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by LinkIsGrim** [comment]
Review when I'm sober.

File: addons/arsenal/CfgGestures.hpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class CfgGesturesMale {
```
Comment (LinkIsGrim) [inline addons/arsenal/CfgGestures.hpp:1]: Move to common

File: addons/arsenal/functions/fnc_showItem.sqf [hunk 2]
```diff
@@ -58,6 +58,7 @@ if (_nextAction != GVAR(currentAction)) then {
     };
 
     if (simulationEnabled GVAR(center)) then {
+        GVAR(center) playActionNow QGVAR(stop);
```
Comment (LinkIsGrim) [inline addons/arsenal/functions/fnc_showItem.sqf:61]:
```suggestion
        GVAR(center) playActionNow QEGVAR(common,stop);
```

**Review by LinkIsGrim — CHANGES_REQUESTED** [review]

File: addons/arsenal/CfgGestures.hpp (hunk 1 above)
Comment (johnb432) [reply addons/arsenal/CfgGestures.hpp]: Done - re-review please.

**Comment by LinkIsGrim** [comment]
I wasn't intending on hijacking this but I got carried away.

**Review by veteran29 — APPROVED** [review]
LGTM

**Review by johnb432 — COMMENTED** [review]
LGTM

**Pull request merged**